sourcegraph/sourcegraph.proto

```diff
@@ -2481,12 +2498,28 @@ service RepoTree {
 // API.
 service Search {
 	// Search searches the full index.
+	// This method is deprecated (use one of the more specific
+	// search methods below)
```

Comment:
Let's use the [Go style for deprecation](https://github.com/golang/go/issues/10909#issuecomment-136492606). Make the first word of the sentence "Deprecated". E.g.:

```Go
// Deprecated: use one of the more specific search methods below.
```